bcgov/HOUS-permit-portal
# Feature/bphh 1288/external api docs

## Description
Adds documentation for external api


## What type of PR is this? (check all applicable)

- [ x] 🍕 Feature

## Related Tickets & Documents
https://hous-bssb.atlassian.net/browse/BPHH-1288


## Steps to QA
- Run migrations and bundle install
- run `bundle exec rspec ./spec/requests/external_api/v1/permit_applications_spec.rb` for specs
- Visit `/integrations/api_docs`
- Should be able to view the docs pag
<img width="1694" alt="Screenshot 2024-05-28 at 4 13 35 PM" src="https://github.com/bcgov/HOUS-permit-portal/assets/32022335/c0625843-0cd8-443f-b195-ce46c9eb6ecc">
